src/javascript/app_2/Stores/Modules/Trading/Helpers/contract_type.js

```diff
@@ -306,6 +317,7 @@ const ContractType = (() => {
         getBarriers,
         getContractType,
         getContractValues,
+        getContractsForThisDuration,
```

Comment:
for better sorting better to move `getContractsForThisDuration` before `getContractType`